mkdocs.en.yml

```
@@ -0,0 +1,186 @@
+repo_name: QuantumNous/new-api
+repo_url: https://github.com/QuantumNous/new-api
+
+theme:
+  name: material
+  language: en
+  custom_dir: newapi/overrides
+  features:
+    - announce.dismiss
+    - content.action.edit
+    - content.action.view
+    - content.code.annotate 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)